lib/moonshot/commands/interactive_command.rb

```diff
@@ -0,0 +1,13 @@
+module Moonshot
+  module Commands
+    class InteractiveCommand < Moonshot::Command
```

Comment:
I feel like this should be a module rather than a class. This would allow commands to import various features, though we may only have interactive for now, without having to pick one to extend from or implementing an ever increasingly complex set of base classes.